### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by daneleven**
Hudson Rock is a cybercrime intelligence company focused on providing actionable intelligence from Infostealer infections.

Hudson Rock offers free cybercrime intelligence API calls (free tier) using a free key, supporting use cases such as:

End User Protection - search an email and discover whether it is associated with a computer that was infected by an Infostealer

External Attack Surface Discovery - search a domain, discover URLs from actual Infostealer infection

Assets Intelligence - search a technology (vpn, confluence, moveit, sharepoint) and discover companies using these technologies, and in which URLs

Domain Intelligence - search a domain and discover compromised employees/customers that were infected by Infostealers

The documentation and free key can be found here: www.cavalier.hudsonrock.com/docs

## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [ ] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -644,6 +644,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [GitGuardian](https://www.gitguardian.com) — Keep secrets out of your source code with automated secrets detection and remediation. Scan your git repos for 350+ types of secrets and sensitive files – Free for individuals and teams of 25 developers or less.
   * [Have I been pwned?](https://haveibeenpwned.com) — REST API for fetching the information on the breaches.
   * [hostedscan.com](https://hostedscan.com) — Online vulnerability scanner for web applications, servers, and networks. Ten free scans per month.
+  * [hudsonrock.com](https://cavalier.hudsonrock.com/docs) — Free cybercrime intelligence API calls (free tier) using a free key, supporting use cases such as End User Protection, External Attack Surface Discovery, Assets Intelligence and Domain Intelligence.
```
Comment (ripienaar) [inline README.md:647]: Please link to a product page that details features and pricing, where the free tier needs to be covered.

Comment (daneleven) [reply README.md:647]: Please check the "Help" section in the link attached (https://cavalier.hudsonrock.com/docs) to view pricing.

Comment (ripienaar) [reply README.md:647]: This does not give details of what can be done with free api keys - limits etc.

Comment (daneleven) [reply README.md:647]: Each endpoint in this page has a detailed limits and features.

Comment (ripienaar) [reply README.md:647]: I am not studying every end point of your entry to find out what is offered, what is free and so forth.

There are 1000+ entries here over 2000 PRs, I cannot realistically do that.

Your service requires a clear place to look out for the information allowing not just us but also end users to evaluate what is offered

Comment (daneleven) [reply README.md:647]: All endpoints are free with limited viewing, each endpoint has the explanation of what it does. rate limiting is under "Getting Started".

Comment (ripienaar) [reply README.md:647]: And where to get a free api key? Documentation states to use your own if not testing?

Sorry, I just dont have time to get this information slowly and piece by piece.

The PR template states ``` Pricing information is clearly visible without signup or phone calls``` which this is not.

Feel free to open again once you create a pricing page somewhere with accessible details.

**Pull request closed without merging**